packages/protocol/contracts-0.8/common/EpochManager.sol

```diff
@@ -326,6 +326,18 @@ contract EpochManager is
     );
   }
 
+  /**
+   * @notice Returns the epoch info of a specified epoch.
+   * @param epochNumber Epoch number where epoch info is retreived.
+   * @return Epoch info.
+   */
+  function getEpochInfoOfEpoch(
```

Comment:
Naming opinion: personally think `getEpochInfo` would be clear enough, but up to you.

Reply:
Actually, since this returns the same data as `getCurrentEpoch` (without `Info`), maybe for consistency rename this to just `getEpoch` (or `getEpochByNumber`), and reimplement `getCurrentEpoch` to call this for deduplication.